distr_tracing/distr_tracing_tempo/distr-tracing-tempo-updating.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="dist-tracing-tempo-updating"]
+= Updating distributed tracing platform Tempo
+include::_attributes/common-attributes.adoc[]
+:context: dist-tracing-tempo-updating
+
+toc::[]
+
+Operator Lifecycle Manager (OLM) controls the installation, upgrade, and role-based access control (RBAC) of Operators in a cluster. The OLM runs by default in {product-title}.
+OLM queries for available Operators as well as upgrades for installed Operators.
+For more information about how {product-title} handles upgrades, see the xref:../../operators/understanding/olm/olm-understanding-olm.adoc#olm-understanding-olm[Operator Lifecycle Manager] documentation.
+
+During an update, the {DTProductName} Operators upgrade the managed {DTShortName} instances to the version associated with the Operator. Whenever a new version of the {TempoName} Operator is installed, all the {TempoShortName} application instances managed by the Operator are upgraded to the Operator's version. For example, after upgrading the Operator from 1.10 installed to 1.11, the Operator scans for running {TempoShortName} instances and upgrades them to 1.11 as well.
+
+For specific instructions on how to update the OpenShift Elasticsearch Operator, see xref:../../logging/cluster-logging-upgrading.adoc#cluster-logging-upgrading_cluster-logging-upgrading[Updating OpenShift Logging].
+
+include::modules/distr-tracing-tempo-change-operator-20.adoc[leveloffset=+1]
```

Comment:
I think we will need to remove the included file because it was for Jaeger (I renamed the file name from jaeger to tempo when I was copying all Jaeger files for initial reuse for the topics about Tempo.